src/socket.cpp

```diff
@@ -25,6 +25,13 @@
 #include "socket.h"
 #include "server.h"
 
+#ifdef _WIN32
```

Comment:
None of this should be in the `.cpp` file.  There's a `socket.h` file for socket-related includes.

We should only use Qt network support, where possible, also.